Cargo.toml

```diff
@@ -62,6 +62,7 @@ im-rc = "15.1.0"
 
 [build-dependencies]
 lalrpop = "0.20"
+chrono = { version = "0.4", default-features = false, features = ["now"] }
```

Comment:
I disabled the default feature set because it will otherwise import a complex set of dependencies due to the `locale` feature.